plc4j/drivers/modbus/src/test/java/org/apache/plc4x/java/modbus/base/optimizer/LittleEndianByteSwapTest.java

```diff
@@ -183,9 +184,9 @@ public void testLittleEndianByteSwap() throws Exception {
         input.put("variable129",   new String[]{"coil:165:BOOL",             "false"});
         input.put("variable130",   new String[]{"coil:167:BOOL",             "false"});
         input.put("variable131",   new String[]{"coil:169:BOOL",             "false"});
-        input.put("variable132",   new String[]{"coil:171:BOOL",             "false"});
+        input.put("variable132",   new String[]{"coil:171:BOOL",             "true"});
```

Comment:
Why change this? This should not matter at all when reading and just requires me to change the program on my test-devices.